target/linux/ramips/dts/mt7621_glinet_gl-mt1300.dts

```diff
@@ -67,7 +67,7 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <80000000>;
+		spi-max-frequency = <25000000>;
 		m25p,fast-read;
```

Comment:
For mt7621, 50MHz (actually 44 MHz) is usually a very reliable value. And `m25p,fast-read;` can be removed when frequency lower than 50MHz.

Reply:
As suggested, selected "50MHz" (222MHz/8) and removed the fast-read attribute.  Still seems to work fine on my hardware, but we don't have quite the breadth of test of other people for whom "25MHz" worked well.  Thanks for feedback.